public/src/components/common/button_objs.jsx

```diff
@@ -112,6 +112,7 @@ export default {
         size: 'xsmall',
         style: 'success',
         handler: () => {
+          // this will need to set a dummy user_sid in the state
```

Comment:
Um no it doesn't. Joining a campaign should not set any call SID. Also we decided against putting call SIDs in the client store.

Reply:
✔️